etc/kayobe/inventory/group_vars/controllers/firewall

```diff
@@ -0,0 +1,375 @@
+    comment: "hint: used by 'haproxy' for grafana_server_external"
+  - interface: "{{ public_net_interface }}"
+    port: 5000
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for keystone_external"
+  - interface: "{{ public_net_interface }}"
+    port: 5601
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for kibana_external"
+  - interface: "{{ public_net_interface }}"
+    port: 6080
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_novncproxy_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8000
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for heat_api_cfn_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8004
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for heat_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8774
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8775
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_metadata_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8776
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for cinder_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8780
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for placement_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8786
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for manila_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 9093
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for prometheus_alertmanager_external"
+  - interface: "{{ public_net_interface }}"
+    port: 9292
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for glance_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 9511
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for magnum_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 9696
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for neutron_server_external"
+  - interface: "all"
+    port: 111
+    proto: udp
+    comment: "hint: used by 'systemd'"
```

Comment:
all seems wrong here? I guess I duno what it is doing.

Maybe this should be localhost / lo only?

Reply:
This is the NFS portmapper service. I guess we may want this on the controllers for manilla shares, but not on any other host?